store/tikv/region_request_test.go

```diff
@@ -331,6 +331,18 @@ func (s *mockTikvGrpcServer) RawBatchScan(context.Context, *kvrpcpb.RawBatchScan
 func (s *mockTikvGrpcServer) UnsafeDestroyRange(context.Context, *kvrpcpb.UnsafeDestroyRangeRequest) (*kvrpcpb.UnsafeDestroyRangeResponse, error) {
 	return nil, errors.New("unreachable")
 }
+func (s *mockTikvGrpcServer) RegisterLockObserver(context.Context, *kvrpcpb.RegisterLockObserverRequest) (*kvrpcpb.RegisterLockObserverResponse, error) {
```

Comment:
Prefer to compound the service interface into `mockTikvGrpcServer` struct directly avoid these boilerplate code.
e.g
```go
type mockTikvGrpcServer struct {TiKVService}
```
and remove all `unreachable` methods.

Reply:
I think it's not very proper to do it in this PR.